src/components/FilePreview/Base.vue

```diff
@@ -1,13 +1,7 @@
 <script>
-import { PDF, IMG } from './Types'
 import { getComponent } from '../../lib/file_preview'
```

Comment:
Can we move this (`lib/file_preview`) under components/FilePreview/index (as a default export)? To clean things up a bit.